pennylane_lightning/src/Gates.cpp

```diff
@@ -237,6 +245,13 @@ Pennylane::RotationXGate::RotationXGate(double rotationAngle)
       js, c }
 {}
 
+const double Pennylane::RotationXGate::generatorScalingFactor{ -0.5 };
```

Comment:
I still do worry about this number changing in PL and us not noticing, but I guess we'll be able to notice by having good quality integration tests for the gradient.

Reply:
Yep, I'm not sure how this would else be done. I think we could leave it like this, rely on testing, and then perhaps update it in the future if we figure out a better way.

Reply:
Agree, and it's possibly faster to have it hard-coded than passing from the Python frontend